ganttproject/src/main/java/net/sourceforge/ganttproject/GanttProject.java

```diff
@@ -969,20 +967,34 @@ private Unit sendProjectStateLogs() {
           "refid",
           txns
         ));
+        isSendingInProgress.set(true);
```

Comment:
set the guarding flag before you actually send, otherwise there is a chance to set it to true when the response is already received